src/noahsternberg.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function NoahSternbergComponent() {
+
+    const data = [
+        {id : 1},
+        {projectType : 'Deck building'},
+        {projectLengthInDays : 4},
+        {estimatedLaborCost : 1234},
+        {bidPrice : 1500},
+        {accepted : true}
+    ]
+
+    return (
+        <div>
+            <h2> Project Details </h2>
+            <ul>
+                <li>ID : {data[0].id}</li>
+                <li>Project Type : {data[1].projectType}</li>
+                <li>Project Length (days) : {data[2].projectLengthInDays}</li>
+                <li>Labour Costs : {data[3].estimatedLaborCost}</li>
+                <li>Bid Price : {data[4].bidPrice}</li>
+                <li>Accepted : {data[5].accepted}</li>
+            </ul>
+        </div>
+    )
```

Comment:
Your return portion should work but there is a more elegant way to do it. 

<ul>
        {data.map((datapoint) => (
          <li key={datapoint.id}>{datapoint.name}</li>
        ))}
      </ul>

This will allow you to iterate through your data array and create a new list item for each array object.